docs/ADR/0003-single-clipboard-write.md

```
@@ -0,0 +1,127 @@
+
+| Factory                    | Before                                                      | After                                                  |
+| -------------------------- | ----------------------------------------------------------- | ------------------------------------------------------ |
+| `AIAssistantInsertFactory` | Wrote clipboard, executed paste commands                    | Calls `pasteTextFromClipboard()` only                  |
+| `TerminalInsertFactory`    | Wrote clipboard, called `pasteIntoTerminal(terminal, text)` | Calls `pasteIntoTerminal(terminal)` only               |
+| `EditorInsertFactory`      | Called `insertTextAtCursor(editor, text)`                   | Unchanged (editor insert is direct, not via clipboard) |
+
+`_text` is accepted but ignored by AI assistant and terminal insert factories since the text is already on the clipboard.
+
+### 3. Two-delay model
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)